ConfigurationSystem/Client/tests/PathFinderTests.py

```diff
@@ -0,0 +1,56 @@
+""" Unit tests for PathFinder only for functions that I added
```

Comment:
This file should be renamed `Test_PathFinder.py` otherwise it won't be automatically discovered by pytest